src/js/controller/captions.js

```diff
@@ -213,14 +212,14 @@ define([
             return list;
         }
 
-        function _selectDefaultIndex() {
+        function _selectDefaultIndex(tracksAvailable) {
             var captionsMenuIndex = 0;
             var label = _model.get('captionLabel');
 
             // Because there is no explicit track for "Off"
             //  it is the implied zeroth track
             if (label === 'Off') {
-                _model.set('captionsIndex', 0);
+                _model.set('captionsIndex', captionsMenuIndex);
```

Comment:
Minified no difference in size. I prefer the hard 0 so there's no way moving this would introduce a bug.